src/app/layouts/MainContainer.js

```diff
@@ -1,7 +1,48 @@
-export const MainContainer = () => {
+import { gameOptionsView } from '../views/GameOptionsView';
+
+export const MainContainer = (gameMode = undefined) => {
```

Comment:
Powiem Wam, że nie do końca rozumiem co się tutaj dzieje w tym MainContainer. Ale nie będę tego blokował :) 
W tym pierwszym projekcie przymknę na to oko, ale w następnym już nie :) 
Odwaliliście kawał dobrej roboty! 

Na refactor najwyżej przyjdzie czas.